src/cascadia/TerminalApp/App.cpp

```diff
@@ -220,6 +220,25 @@ namespace winrt::TerminalApp::implementation
             const auto& profile = _settings->GetProfiles()[profileIndex];
             auto profileMenuItem = Controls::MenuFlyoutItem{};
 
+            // add the keyboard shortcuts for the first 10 profiles
+            if (profileIndex < 10)
+            {
+                auto profileShortcut = Windows::UI::Xaml::Input::KeyboardAccelerator{};
+                profileShortcut.Modifiers(Windows::System::VirtualKeyModifiers::Control | Windows::System::VirtualKeyModifiers::Shift);
+                
+                if (profileIndex == 9)
+                {
+                    // the last tab is actually 0 in the shortcut, so make sure to handle that
+                    profileShortcut.Key(Windows::System::VirtualKey::Number0);
+                }

```

Comment:
Just a note: this may need to change, because of #826 and #831

Reply:
Yep, on it!  Holding this checking until a few things are in.  Cannot test mapping of keybindings due to profiles being wiped out upon start for keybindings.